uber_sante/controllers/booking_controller.py

```diff
@@ -29,25 +32,28 @@ def book():
     # TODO: connect the call to the scheduler class to try reserving the availability (line 51)
     # TODO: connect the call to the booking_service to create the booking in the Booking table (line 54)
 
-        appointment_id = request.args.get('appointment_id')
+        availability_id = request.args.get('availability_id')
```

Comment:
Before all this, make a check that the user is logged;

![image](https://user-images.githubusercontent.com/22150653/53436088-5a230e00-39c8-11e9-8046-13e06088a7e4.png)

Reply:
What i have posted above is an example of how to check if the user is logged

Reply:
Done